app/javascript/components/subject/MoveToHousehold.js

```diff
@@ -1,69 +1,231 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
-import { Form, Row, Col, Button, Modal } from 'react-bootstrap';
+import { Form, Row, Col, Button, Modal, InputGroup, OverlayTrigger, Tooltip } from 'react-bootstrap';
 import axios from 'axios';
+import moment from 'moment-timezone';
+import _ from 'lodash';
 
-import reportError from '../util/ReportError';
+import BadgeHOH from '../util/BadgeHOH';
+import CustomTable from '../layout/CustomTable';
 
 class MoveToHousehold extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      table: {
+        colData: [
```

Comment:
I actually had a discussion about this with the advanced filter (which does the same exact thing as this, and probably needs to also be changed at some point). Could we move colData outside of state, and make it a constant? I'd imagine it's really supposed to be immutable, correct?

Reply:
Hmm, yeah I guess in all of our use cases so far it has been unchanged, though I'm not sure if that will always be the case in all uses of the CustomTable. I'm fine with extracting it - though I would say we use this exact format in `PatientTable`, `AdminTable`, and `AuditModal` so we'd probably want to change it there too for consistency if so. Thoughts?

Reply:
eh, it's nit-picky. Worth opening a ticket for though.